stingray/covariancespectrum.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import division
+
+import numpy as np
+
+from stingray import Lightcurve
+import stingray.utils as utils
+
+
+class Covariancespectrum(object):
+
+    def __init__(self, event_list, bin_width, band_interest=None,
+                 ref_band_interest=None):
+        """
+        Parameters
+        ----------
+        event_list : numpy 2D array
+            A numpy 2D array with first column as time of arrival and second
+            column as photon energies associated.
+
+        bin_width : float
+            The width of the energy bin when the spectrum is divided into N
+            divisions.
+
```

Comment:
I would make `bin_width` a keyword parameter with default `None`, in which case the energies will be used as is (one band per energy). I think otherwise this should be a list with min/max values for each energy band of interest, to allow for energy bands that don't all have the same width.